config/cola-elmo.conf

```diff
@@ -0,0 +1,51 @@
+// Run elmo model on cola-analysis dataset 
```

Comment:
Two extra suggestions:
- Move this to a cola directory, with a readme explaining that this is for this seminar. These configs are probably useful to others, but they're distinctive enough that we should probably keep them separate.
- Proofread for typos. Epoches => epochs. numberof => number of.

Reply:
'Run' suggests just evaluating. => Train and run.